### PR TITLE
Audit AJAX method

### PR DESCRIPTION
**Pull request by bordoni**
_Ref:_ [C#45484](https://central.tri.be/issues/45484)

### CODE REVIEW

File: lib/tribe-meta-box.php [hunk 1]
```diff
@@ -157,6 +157,18 @@ public function delete_file() {
 			die( '1' );
 		}
 
+		$post = get_post( $post_id );
+		if ( ! $post instanceof WP_Post ) {
+			die( '1' );
+		}
+
+		$cap = 'edit_' . $post->post_type;
```
Comment (barryhughes) [inline lib/tribe-meta-box.php]: The general pattern is `edit_{post_type}` but not always (and APM may be used for all kinds of post types we don't know about) ... might something along the lines of `get_post_type_object( $post->post_type )->cap->edit_posts` be a safer bet?

File: lib/tribe-meta-box.php [hunk 2]
```diff
@@ -174,6 +186,21 @@ public function reorder_images() {
...
```
Comment (barryhughes) [inline lib/tribe-meta-box.php]: As above, may not be the best way to determine the 'edit post' capability for whatever the current post type is.

File: lib/tribe-meta-box.php (hunk 1 above)
Comment (bordoni) [inline lib/tribe-meta-box.php]: Right, I will review this.

**Comment by bordoni** [comment]
Updated.

**Comment by barryhughes** [comment]
yabba

**Pull request merged**